util/reggen/gen_dv.py

```diff
@@ -41,6 +41,11 @@ def sv_base_addr(top: Top, inst_name: str):
                              top.by_inst_name[inst_name][0])
 
 
+def is_top_inst(b):
```

Comment:
btw, i'm not sure you want to re-use it, but there is a function that the design side employs for this.
[see here](https://github.com/lowRISC/opentitan/blob/master/util/topgen/lib.py#L305)

up to you though.

Reply:
Thanks for the suggestion.
For `is_inst(module)`, its input is module. for this function, input is a block which is an instance of module. module has "type", but block doesn't.

perhaps create a new function for DV.